### PR TITLE
@W-9806702 Allow UIScrollable class in Android UIAutomator locator

### PR DESCRIPTION
**Pull request by rajukamal**
@helenren @lizaiv77 could you please review? This is required to use Android UIAutomator locator with 'UIScrollable' class to scroll to an element that is not in view.

### CODE REVIEW

File: utam-core/src/main/java/utam/core/selenium/appium/LocatorUIAutomator.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 public class LocatorUIAutomator extends LocatorBy {
 
   public static final String UI_AUTOMATOR_SELECTOR_PREFIX = "new UiSelector().";
+  public static final String UI_AUTOMATOR_SCROLLABLE_PREFIX = "new UiScrollable";
```
[the remark this answers is not in the document]
Comment (lizaiv77) [reply utam-core/src/main/java/utam/core/selenium/appium/LocatorUIAutomator.java:18]: hm, what tests outside package?
it's been added in this PR and used in tests from same package. i actually tried to change access before asking

**Review by lizaiv77 — CHANGES_REQUESTED** [review]
same request - static variable should have package access

File: utam-core/src/main/java/utam/core/selenium/appium/LocatorUIAutomator.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (rajukamal) [reply utam-core/src/main/java/utam/core/selenium/appium/LocatorUIAutomator.java]: we no longer are prepending the locator with new UiSelector() and won't need these constants anymore.

**Comment by rajukamal** [comment]
@helenren @lizaiv77 Appium allows usage of Ui Automator locators without 'new UiSelector' prefix. So have removed the code to add the prefix. Now we support both formats - with and without prefix. 
Above change has been done in addition to the support for locators with prefix 'new UiScrollable' to scroll elements into view. 
Additionally, we now are supporting 'resourceIdMatches' method.

**Comment by helenren** [comment]
LGTM. thanks. @rajukamal

**Pull request merged**